Library/Homebrew/utils.rb

```diff
@@ -188,14 +188,32 @@ def self.homebrew_version_string
 
   def self.install_gem_setup_path!(gem, version = nil, executable = gem)
     require "rubygems"
-    ENV["PATH"] = "#{Gem.user_dir}/bin:#{ENV["PATH"]}"
 
-    args = [gem]
-    args << "-v" << version if version
+    if Gem::Specification.find_all_by_name(gem, version).empty?
+      ohai "Installing missing or updating outdated '#{gem}' gem"
```

Comment:
This is gonna be a lot more noisy if it's printed for every gem. I think we can perhaps just omit this?